### ISSUE TITLE
JupyterHub chat channels: Gitter or Zulip?

### ISSUE THREAD

**Issue by manics** [jupyterhub/team-compass#751]
A while ago there was a Jupyter discussion on moving away from Gitter to [Zulip](https://jupyter.zulipchat.com/) or Discord. After a pause there's a new spurt of activity around Zulip:
https://github.com/jupyter/governance/issues/182#issuecomment-2489318648 and subsequent posts.

What do people think? Switch to Zulip, stick with Gitter, or wait and see? I don't think we should support multiple chat platforms.

**Comment by minrk** [comment]
Zulip seems to be where folks are going. It would be nice to move away from the gitter spam.

**Comment by choldgraf** [comment]
+1 from me on Zulip. I'm more familiar with discord but I know the nepari folks have used Zulip for a while and really like it.

**Comment by choldgraf** [comment]
To clarify: is the plan to use the jupyter Zulip and create a channel or something? Or to set up a dedicated hub team Zulip? I think I have a preference for the former in order to encourage more cross project communication, but if folks feel strongly about the latter then I'm ok with that

**Comment by minrk** [comment]
I would imagine channel/room(s) in the Jupyter Zulip

**Comment by manics** [comment]
The existing Jupyter Zulip https://jupyter.zulipchat.com/

**Comment by sgibson91** [comment]
Yeah, using gutter was always a big barrier for me, that got worse when it officially became a matrix instance. I'll go wherever, so long as it's away from gitter 😆

**Comment by rgaiacs** [comment]
> I would imagine channel/room(s) in the Jupyter Zulip

This is what makes more sense to me.

**Comment by choldgraf** [comment]
### Try out the `#jupyterhub` zulip channel!

I've asked on the Jupyter Zulip instance, and they've [created a `#jupyterhub` channel](https://jupyter.zulipchat.com/#narrow/channel/469744-jupyterhub/) for us to try out. We don't have to treat this as a "decision" but I thought it'd be helpful to create something so that people could get a feel for what the workflow would look like.

**Comment by consideRatio** [comment]
I'm very positive towards Zulip after having used it a while now!

**Comment by yuvipanda** [comment]
my own involvement in some of these projects is now purely *because* of the interactions in Zulip, so it's now a big win.

**Comment by sgibson91** [comment]
I feel much more able to participate in chats because of zulip. Gitter wasn't great, and the experience for me got worse when it became "just another matrix server".

**Comment by minrk** [comment]
Seems very positive so far, I opened #757 to udpat ethe links in this repo, then we can update the Gitter room descriptions to redirect folks

**Comment by sgibson91** [comment]
Merged!

**Comment by minrk** [comment]
Thanks! I added links to the jupyterhub rooms and made them invite-only so new people (i.e. spam bots) can't join and post new messages. The history is still fully public and anyone already in there can still post messages.

Notably, the 'Binder' room isn't actually a gitter room, it's on petrichor.me, so I'm not sure who actually has admin rights to change anything about it. I posted a message there, though, since that's all I can do.

**Comment by sgibson91** [comment]
Ah, that's my friend Jez Cope. Stuff with that room got all messed up in the switch from Gitter to Matrix and the support team did not come back with anything useful. I've pinged him in the OLS/OpenSeeds Slack (because I, embarrassingly cannot remember his GitHub handle!) and asked for him to follow your lead.

**Comment by sgibson91** [comment]
I also see this room too? https://matrix.to/#/#jupyterhub_binder:gitter.im

**Comment by jezcope** [comment]
Oh yeah, things got a bit messed up when New Vector acquired Gitter, the transition was not handled well. I no longer have access to an account with full admin rights to that room so I can't change permissions to prevent further posting, but I have:

- Removed all remaining room addresses that I could (including the main one, @binder:petrichor.me)
- Made it invite-only
- Pinned @minrk's message
- Updated the room topic with a link to Zulip

> I also see this room too? https://matrix.to/#/#jupyterhub_binder:gitter.im

This address seems to resolve to the same room, so I think we're good. Let me know if there's anything else I can do!

**Comment by jezcope** [comment]
Oh, the other thing that I can do if you want is to kick all existing users from the room and then leave, which would functionally delete it since it's invite-only. It would live on for a while in the databases of servers that participated in it but would be unjoinable and eventually be garbage-collected. But there might be some value in retaining access to its history.

**Comment by minrk** [comment]
Wonderful, thank you! I don't feel the need to kick anyone out. The only real reason I turned on invite-only is to stop having to deal with the spam bots, which seem to be most new joins.

**Comment by sgibson91** [comment]
I also wrote a super-quick blog post announcing this decision as well https://blog.jupyter.org/join-jupyterhub-on-zulip-chat-4c17491bd96d

**Comment by minrk** [comment]
Great, thank you!

**Comment by jezcope** [comment]
> I also wrote a super-quick blog post announcing this decision as well https://blog.jupyter.org/join-jupyterhub-on-zulip-chat-4c17491bd96d

@sgibson91 Is this published yet? It sends me to a Medium login page...

**Comment by sgibson91** [comment]
@jezcope no it's not. That link is to a draft. The folks who monitor the Jupyter blog tell me they go through it weekly on Thursdays, so I expect it to be up later today or tomorrow.
